larky/src/main/java/com/verygood/security/larky/modules/AccountUpdaterModule.java

```diff
@@ -0,0 +1,123 @@
+package com.verygood.security.larky.modules;
+
+import com.google.common.collect.ImmutableList;
+import com.verygood.security.larky.modules.vgs.aus.LarkyAccountUpdater;
+import com.verygood.security.larky.modules.vgs.aus.MockAccountUpdaterService;
+import com.verygood.security.larky.modules.vgs.aus.NoopAccountUpdaterService;
+import com.verygood.security.larky.modules.vgs.aus.spi.AccountUpdaterService;
+import java.util.List;
+import java.util.ServiceLoader;
+import net.starlark.java.annot.Param;
+import net.starlark.java.annot.ParamType;
+import net.starlark.java.annot.StarlarkBuiltin;
+import net.starlark.java.annot.StarlarkMethod;
+import net.starlark.java.eval.*;
+
+@StarlarkBuiltin(
+    name = "native_au",
+    category = "BUILTIN",
+    doc = "Overridable Account Updater API in Larky")
+public class AccountUpdaterModule implements LarkyAccountUpdater {
+  public static final AccountUpdaterModule INSTANCE = new AccountUpdaterModule();
+
+  public static final String ENABLE_MOCK_PROPERTY =
+      "larky.modules.vgs.nts.enableMockAccountUpdater";
+
+  private final AccountUpdaterService AccountUpdaterService;
+
+  public AccountUpdaterModule() {
+    final ServiceLoader<AccountUpdaterService> loader =
+        ServiceLoader.load(AccountUpdaterService.class);
+    final List<AccountUpdaterService> AccountUpdaterProviders =
+        ImmutableList.copyOf(loader.iterator());
+
+    if (Boolean.getBoolean(ENABLE_MOCK_PROPERTY)) {
+      AccountUpdaterService = new MockAccountUpdaterService();
+    } else if (AccountUpdaterProviders.isEmpty()) {
+      AccountUpdaterService = new NoopAccountUpdaterService();
+    } else {
+      if (AccountUpdaterProviders.size() != 1) {
+        throw new IllegalArgumentException(
+            String.format(
+                "AccountUpdaterModule expecting only 1 network token provider of type AccountUpdaterService, found %d",
+                AccountUpdaterProviders.size()));
+      }
+      AccountUpdaterService = AccountUpdaterProviders.get(0);
+    }
+  }
+
+  @StarlarkMethod(
+      name = "lookup_card",
```

Comment:
should this change to lookup PAN instead of lookup card to match the documentation?